src/main/java/seedu/address/model/person/PersonComparator.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.person;
```

Comment:
Should the comparator be in the person directory? Or can we consider putting it in the model instead, as having the comparator in the person directory seems like the comparator is part of the person.